tools/benchmark/metrics/prepare_metrics.sh

```diff
@@ -12,6 +12,18 @@ MYDIR=$(dirname $(realpath "$0"))
 main() {
   cd "${MYDIR}/../../../third_party"
   local zipurl
+  local repourl
+  for repourl in \
+    'https://github.com/veluca93/IQA-optimization.git'
+    'https://github.com/Netflix/vmaf.git'
+	  'https://github.com/thorfdbg/difftest_ng.git'
+  do
+    local reponame=$(basename "${repourl}")
+    local dirname=$(basename "${reponame}")
+    if [[ ! -e "${dirname}" ]]; then
+      git clone "${repourl}"
```

Comment:
Should we perhaps check out a specific revision, and clone it shallowly if applicable?

Reply:
Nah, too much effort...

Reply:
so lazy!